### PR TITLE
Fix team alignment

### LINKED ISSUE #5: Use Bootstrap for a responsive page layout
This would really help with structuring our team listing into a grid as our number of staff continues to grow. See screenshots of this section currently without a fixed grid.

No. 1: There is a wide gap between Motun and San.
screenshot 2019-01-12 17 36 15
No. 2: The columns are not aligned.
screenshot 2019-01-12 17 36 38

### PR DESCRIPTION
**Pull request by gkang2018**
Added layout to team section, Closes #5

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -31,11 +31,11 @@
     .tile > #projects > table p { margin:2px 0px 2px 0px; }
     .tile > .people {
       margin-top:75px;
-      text-align:justify; -ms-text-justify:distribute-all-lines; text-justify:distribute-all-lines;
+      text-align:left; -ms-text-justify:distribute-all-lines; text-justify:distribute-all-lines;
       line-height:1.45; letter-spacing:0.01em; text-rendering:optimizeLegibility; font-feature-settings:"kern"; -webkit-font-feature-settings:"kern"; -moz-font-feature-settings:"kern"; -moz-font-feature-settings:"kern=1";
       cursor:default;
     }
-    .tile > .people > div { display:inline-block; *display:inline; zoom:1; margin:0px 0px 20px 0px; padding:0px 5px 0px 5px; }
+    .tile > .people > div { display:inline-block; *display:inline; zoom:1; margin:0px 0px 20px 0px; padding:0px 5px 0px 5px; width: 150px; max-width: 200px; word-wrap: break-word;;}
```
Comment (frederickjansen) [inline css/style.css:38]: Double ;; at the end

File: css/style.css [hunk 2]
```diff
@@ -31,11 +31,11 @@
     .tile > #projects > table p { margin:2px 0px 2px 0px; }
     .tile > .people {
       margin-top:75px;
-      text-align:justify; -ms-text-justify:distribute-all-lines; text-justify:distribute-all-lines;
+      text-align:left; -ms-text-justify:distribute-all-lines; text-justify:distribute-all-lines;
```
Comment (frederickjansen) [inline css/style.css:34]: Does text-align left conflict with the other alignment styles? Don't think -ms-text-justify and text-justify are still needed with align left.

File: index.html [hunk 3]
```diff
@@ -17,6 +17,7 @@
   ga('create', 'UA-89848325-1', 'auto');ga('send', 'pageview');
   </script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
```
Comment (frederickjansen) [inline index.html:20]: It feels like a little overkill to add Bootstrap for just one class. Can we just use the CSS of "row" and add it to our stylesheet instead?

**Comment by gkang2018** [comment]
alignmentSmallScreen
alignmentHalfScreen
alignmentFullScreen

**Comment by gkang2018** [comment]
Here are three screenshots showing the alignment when I change the screen size. 


alignmentSmallScreen
alignmentHalfScreen
alignmentFullScreen

**Comment by frederickjansen** [comment]
Looks great, thanks!

**Pull request merged**